pkg/devfile/parser/data/common/command_helper.go

```diff
@@ -0,0 +1,71 @@
+package common
+
+import (
+	"strings"
+
+	v1 "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+)
+
+// GetID returns the ID of the command
+func GetID(dc v1.Command) string {
+	return dc.Id
+}
+
+// SetIDToLower converts the command's id to lower case for more efficient processing and returns the new id
+func SetIDToLower(dc *v1.Command) string {
+	var newId string
+	newId = strings.ToLower(dc.Id)
+	dc.Id = newId
+	return newId
+}
```

Comment:
What's the difference between this and just inlining
```go
command.Id = strings.ToLower(command.Id)
```
where we would use this function?

Reply:
I think you are right, except for they function has a return value which returning the new command.Id after setting it. 
this is how we use it in our code: https://github.com/devfile/parser/blob/3a759b5e0196f8cb5908054a83aa0537773874bf/pkg/testingutil/devfile.go#L84
But I'm fine with removing this function.